freerdp.yaml

```diff
@@ -27,12 +27,13 @@ environment:
       - libxext-dev
       - libxi-dev
       - libxinerama-dev
-      # - libxkbfile-dev
       - libxkb-dev
       - libxkbcommon-dev
       - libxrender-dev
       - libxv-dev
       - linux-headers
+      # - libxkbfile-dev
```

Comment:
Do we still need this if it's commented out?

Reply:
i have no idea, but yam moved it. Imho commented out stuff is never needed as git history exists.